lib/elixir/src/elixir_with.erl

```diff
@@ -1,7 +1,7 @@
 -module(elixir_with).
 -export([expand/3, translate/3]).
 -include("elixir.hrl").
-
+-define(ref(Meta), {ref, Meta, elixir_with}).
```

Comment:
`If I have an with/else inside another with/else, could those variables conflict?` =>

Works, as expected, but seems, like `:ok` | `:error` can be less confusing/questionable implementation.

```elixir
with {:ok, width} <- Map.fetch(opts, :width),
     {:ok, height} <- (with {:ok, height} <- Map.fetch(opts, :height) do
                            {:ok, height} 
                       else 
                         :error -> :exp1
                       end) do 
  {:ok, width * height}
else
  :error -> :exp2
  :exp1  -> :exp1
end
```